.github/requirements-cicd-gpu.txt

```diff
@@ -1,6 +1,6 @@
 https://github.com/jina-ai/jina/archive/refs/heads/master.zip
-tensorflow==2.5.0
-paddlepaddle-gpu
-torch
-torchvision
-scipy
\ No newline at end of file
+tensorflow<=2.5.2
```

Comment:
I would still pin the exact version. What is the advantage of doing `<=` instead of `==`?

Reply:
And maybe we can pin it to 2.6.1, and then pin `keras` as well - that would prevent the issue we had